docs/contracts/events.rst

```diff
@@ -160,6 +160,6 @@ where the long hexadecimal number is equal to
 Additional Resources for Understanding Events
 ==============================================
 
-- `Javascript documentation <https://github.com/ethereum/wiki/wiki/JavaScript-API#contract-events>`_
-- `Example usage of events <https://github.com/debris/smart-exchange/blob/master/lib/contracts/SmartExchange.sol>`_
-- `How to access them in js <https://github.com/debris/smart-exchange/blob/master/lib/exchange_transactions.js>`_
+- `Javascript documentation <https://github.com/ethereum/web3.js/blob/1.x/docs/web3-eth-contract.rst#events>`_
```

Comment:
The best link for this would be https://web3js.readthedocs.io/en/v1.2.11/web3-eth-contract.html but then I'd have to link to a specific version becuause replacing `v1.2.11` with [`latest`](https://web3js.readthedocs.io/en/latest/web3-eth-contract.html) in the URL does not work. [`1.0`](https://web3js.readthedocs.io/en/1.0/web3-eth-contract.html) kinda works but redirects to `v1.2.0` rather than the latest 1.x version. So I'm just linking to directly to this file in the repo.

I could also link to 2.x but there are no stable 2.x releases yet so I don't think it's a good idea.